tools/dockerfile/distribtest/ruby_jessie_x64/Dockerfile

```diff
@@ -16,4 +16,4 @@ FROM debian:jessie
 
 RUN apt-get update && apt-get install -y ruby-full
 
-RUN gem install bundler
+RUN gem install bundler -v 1.17.3 --no-document
```

Comment:
if @apolcyn  is happy with this I'm fine too, but note that  this is a distrib test so ideally things would work with a vanilla installation of whatever system is being under tests (here we are trying to test if one can use ruby under debian jessie) - so requiring special tweaks in the distribtest dockerfiles also means users will need to use the same tweaks to be able to use grpc (which kind of beats the purpose of a distribtest).